examples/modules/async-number/binding.gyp

```diff
@@ -0,0 +1,39 @@
+{
+  "variables": {
+    "napajs_root": "<!(node -e \" \
+      var path = require('path'); \
+      process.stdout.write(path.resolve(path.dirname(require.resolve('napajs')), '..')); \
```

Comment:
So, have you decided not to introduce method to return inc/bin folder from 'napajs' module?